### PR TITLE
New feature - Import external and Built-in layers

### PR DESCRIPTION
**Pull request by Ebubeker**
This feature allows you to grab your own layers from an endpoint that you provide. It also allows you to import built in layers that aren't part of the starter project.

## Description
Fixed the legend issue a bit. Still have some more things to add there.

## Review
This pull request is created to be reviewed before adding more stuff to the feature

### CODE REVIEW

File: app/client/src/plugins/vuetify.js [hunk 1]
```diff
@@ -1,8 +1,4 @@
-if (process.env.VUE_APP_FONTAWESOME_NPM_AUTH_TOKEN) {
-  require("@fortawesome/fontawesome-pro/css/all.css");
-} else {
-  require("@fortawesome/fontawesome-free/css/all.css");
-}
```
Comment (majkshkurti) [inline app/client/src/plugins/vuetify.js:5]: Maybe we can revert this to how it was using font-awesome pro.

**Review by majkshkurti — COMMENTED** [review]
Thank you @Ebubeker. I changed the branch to merge into the 'dev' instead..

File: app/client/src/plugins/vuetify.js (hunk 1 above)
Comment (Ebubeker) [reply app/client/src/plugins/vuetify.js:5]: Yeah sure

File: app/client/src/components/layers/importLayers/ImportExternalLayers.vue [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+              </v-flex>
+            </v-layout>
+          </v-card-text>
+        </div>
+        <div v-if="option === 'upload'">
+          <v-app-bar :color="appColor.primary" dark>
+            <v-app-bar-nav-icon
+              ><v-icon>fas fa-layer-group</v-icon></v-app-bar-nav-icon
+            >
+            <v-toolbar-title>External Geoportal</v-toolbar-title>
```
[the remark this answers is not in the document]
Comment (Ebubeker) [reply app/client/src/components/layers/importLayers/ImportExternalLayers.vue]: I have done them only in english, and I can say that it is finished now, at least this issue

**Comment by Ebubeker** [comment]
Basically there is a new component where I list all of geoportals from a geoserver, and it kinda loads a bit slowly due to the large number of geoportals. Not all geoportals work so I have done a popup that basically reasons why it doesn't work. I kept it simple and clear.

**Pull request merged**